packages/medusa/src/api/routes/admin/sales-channels/index.ts

```diff
@@ -3,7 +3,9 @@ import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
 import "reflect-metadata"
 import { isFeatureFlagEnabled } from "../../../middlewares/feature-flag-enabled"
 import { SalesChannel } from "../../../../models"
-import middlewares from "../../../middlewares"
+import middlewares, { transformBody } from "../../../middlewares"
+import "reflect-metadata"
```

Comment:
comment: Would like to avoid this import, if possible

Reply:
Yes, why do we have that import here?